phenopacketlab-model/src/main/java/org/monarchinitiative/phenopacketlab/model/MinedConcept.java

```diff
@@ -0,0 +1,42 @@
+package org.monarchinitiative.phenopacketlab.model;
+
+public class MinedConcept {
+
+    String id;
+    int start;
+    int end;
+    boolean excluded;
+
+    public MinedConcept(String id, int start, int end, boolean excluded) {
+        this.id = id;
+        this.start = start;
+        this.end = end;
+        this.excluded = excluded;
+    }
+
+    public String getId() {
+        return id;
```

Comment:
Is hashCode and equals needed? I usually use
		```
<!-- https://mvnrepository.com/artifact/uk.co.jemos.podam/podam -->
		<dependency>
			<groupId>uk.co.jemos.podam</groupId>
			<artifactId>podam</artifactId>
			<version>7.2.6.RELEASE</version>
			<scope>test</scope>
		</dependency>
```
Which can autoset random bead properties in my tests. This would pick up if hashCode / equals has not been implemented in the bean. Of course it may not matter depending on how bean used.